components/IngredientCard.tsx

```diff
@@ -143,6 +143,16 @@ const IngredientCards = (props: { ingredients: Ingredient[] }) => {
     }))
   );
 
+  if(cardsToDisplay.length === 0) {
+    return (
+      <Box sx={{ display: "flex", py: 2, justifyContent: "center", alignItems: "center" }}>
+        <Typography level="body5" fontSize="md" textAlign={"center"}>
+          You have not added any ingredients yet!!!
```

Comment:
@stephanegigandet We might need a kind of cleaning about text because even myself ends up doing a mix between french and english

Reply:
Yeah ! Add ingredients button was also English, Rate my recipe is also English, and also many things are English, so i added it in English:(